compiler/tests/testSuite.bal

```diff
@@ -57,15 +68,41 @@ function testCompileEU(string path, string kind) returns file:Error|io:Error? {
     }
 }
 
-function compileErrorLineNumber(CompileError err) returns int? {
-    if err is io:Error {
-        return ();
+function moduleDir(string filePath) returns string|file:Error? {
+    string subModPath = filePath.substring(0, filePath.length()-4) + ".modules";
+    if check file:test(subModPath, file:IS_DIR) {
+        return check file:normalizePath(subModPath, file:CLEAN);
+    }
+}
+
+function expectedErrorLocation(CompileError err, string path) returns [int, string]|file:Error|io:Error? {
+    string? modulePath = check moduleDir(path);
+    string? errorFileName = ();
+    int? errorLineNumber = ();
+    if modulePath is string {
+        foreach var dir in check file:readDir(modulePath) {
+            if !check file:test(dir.absPath, file:IS_DIR) {
+                continue;
+            }
+            foreach var file in check file:readDir(dir.absPath) {
+                [errorLineNumber, errorFileName] = findErrorLine(file.absPath, errorLineNumber);
```

Comment:
This isn't detecting duplicates.

Reply:
`findErrorLine` will check if the given `currentErrorLocation` to be null if it find an error line (this is suppose to detect if multiple files have errors). `errorLine` will check if it already have found an error line every time it come across an error line (this is suppose to detect if there are multiple errors in the same file)